File: LeetCode Solutions/January 2021 LeetCoding Challenge/Check If All 1's Are at Least Length K Places Away/Check If All 1's Are at Least Length K Places Away.java
class Solution {
    public boolean kLengthApart(int[] nums, int k) {
        for (int i=0, count=0; i<nums.length; i++){
            if (nums[i] == 1 && count < k && i != 0){
                return false;
            }
            else if (nums[i] == 1){
                count = 0;
            } else{
                count++;
            }
        }
        return true;
    }
}

File: LeetCode Solutions/January 2021 LeetCoding Challenge/Check If All 1's Are at Least Length K Places Away/Check If All 1's Are at Least Length K Places Away.py
class Solution:
    def kLengthApart(self, nums: List[int], k: int) -> bool:
        count = 0;
        for i in range(len(nums)):
            if (nums[i] == 1 and count < k and i != 0):
                return False
            elif (nums[i] == 1):
                count = 0
            else:
                count += 1
        return True